Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -185,7 +183,6 @@ void EbrApplicationActivated(IActivatedEventArgs^ args) {
 
 void _ApplicationMainLaunch(ActivationType activationType, Platform::Object^ activationArg) {
     _ApplicationLaunch(activationType, activationArg);
-
     _appEvents = ref new AppEventListener();
     _appEvents->_RegisterEventHandlers();
```

Comment:
Can we create/register this in the UIApplicationMain initializer, so we can get rid of _ApplicationMainLaunch and just call _ApplicationLaunch in its place?